.changeset/quick-keys-remain.md

```diff
@@ -0,0 +1,5 @@
+---
+"vercel": minor
```

Comment:
Does this really need to trigger a release? It's only related to tests it looks like.

```suggestion
```